Sources/BrowserServicesKit/Email/EmailManager.swift

```diff
@@ -197,6 +213,11 @@ public class EmailManager {
     }
 
     private var alias: String? {
+        Self.lock.lock()
+        defer {
+            Self.lock.unlock()
+        }
+
```

Comment:
This seems to be the only alias-related logic guarded by lock, is that intended?

Reply:
That was not intended, my idea was to guard all functions that call any storage API, but it looks like I missed `consumeAliasAndReplace` and `fetchAndStoreAlias`. Let me fix it.